Base/Crypto/Crypto.Static/KeyConfirmation.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+namespace Base::KeyConfirmators
+{
+/** @class KeyConfirmation
+* @brief Interface for key confirmation protocol
+* @details Designed for used in Connection class to compare server's and client's keys after generating it
+* by IKeyAgreement
+*/
+template <typename T = std::string>
```

Comment:
Why do you need to make it template class and at the same time you define template param as default one

Reply:
To confirm encryption key we can use any data type with oveloaded ==, so using 'template' gives flexibility in choosing data type.